client/src/modules/journal/journal.js

```diff
@@ -358,15 +358,22 @@ function JournalController(
   }
 
   // display the journal printable report of selected transactions
-  vm.openJournalReport = function openJournalReport() {
+  vm.openJournalReport = function openJournalReport(type) {
```

Comment:
You changed this function to require a type, but in the dropdown menu, you never pass it a type - see Line 44 of the journal.html file.  That may be what is breaking your tests.

Reply:
This function shouldn't take any parameters.  It isn't called with any parameters.